access/templates/access/graded_form.html

```diff
@@ -123,32 +137,37 @@
 			{% endif %}
 
 			{% else %}
+			<!-- Text-field -->
 			{{ field }}
 
 			{% if field.errors %}
 				{% include "access/field_errors.html" with errors=field.errors %}
 			{% endif %}
 
 			{% if field.field.correct %}
+			<!-- Correct answer of a text-field to be shown after max. submissions -->
 				<div data-aplus-once="yes" class="question-feedback">
 					<p class="help-block">
 						{% trans "Correct answer" %}:
 						{{ field.field.correct }}
 					</p>
 				</div>
+			<!-- -->
 			{% endif %}
 
+			<!-- -->
 			{% endif %}
 
-
 		{% if result.accepted and not field.field.row_label or field.field.close_table %}
+		<!-- Question feedback: correctness and hints -->
 			<div class="question-feedback">
 				{% if field.field.type == "checkbox" %}
 					{% include "access/graded_form_feedback.html" with hints=field.field.hints|find_common_hints:field.value answer_correct=field.field.answer_correct %}
 				{% else %}
 					{% include "access/graded_form_feedback.html" with hints=field.field.hints answer_correct=field.field.answer_correct %}
 				{% endif %}
 			</div>
+		<!---->
```

Comment:
I think the end comments would be more useful if they included a reference to what they are ending (e.g. `<!-- / Question feedback -->` or `<!-- end of question feedback -->`